README.md

```diff
@@ -1,43 +1,60 @@
-# Welcome to the trusted output app
+# Welcome to the Trusted Output App
 
 ## This application is responsible for:
 
 - Receiving a CSV file from the research container with the results of the code job
 - Saving that CSV to memory (TBD) for the lifespan of this container
-- Allowing members(TM) to log in and approve the results of the code job (AKA the CSV files)
-- Once the members approve the results, this application will post the CSV file to the management app (Endpoint TBD)
+- Setting up results to be reviewed and sent to the Management App in one of two ways
 
-## Hitting the upload endpoint:
+### Option 1: Encrypt results and send to Management App
 
-```
-curl -X POST http://localhost:3002/api/job/:jobId/upload -F "file=@test-data/industry.csv" -u admin:password
-curl -X POST http://localhost:3002/api/job/:jobId/upload -F "file=@test-data/currency.csv" -u admin:password
-```
+This happens if there are Member Reviewer public keys (from the Member Reviewer Keypair) in the Management App's database.
```

Comment:
is there a spot i can link to the API docs?

Reply:
maybe we can add it in here: https://github.com/safeinsights/knowledge-base/blob/main/SafeInsights-Inter%E2%80%90App-Communication-API.md? under the fingerprint and key and api security portion?